src/ast/SHOW.ts

```diff
@@ -0,0 +1,31 @@
+import NODE from "./NODE";
+import SELECTED_CARDS from "./SELECTED_CARDS";
+import STAT_TO_SHOW from "./STAT_TO_SHOW";
+import * as constants from "../lib/constants";
+
+export default class SHOW extends NODE {
+  showSubject: SELECTED_CARDS | STAT_TO_SHOW | null = null;
+  limit: number = NaN;
+
+  parse() {
+    this.tokenizer.getAndCheckToken("Show stats for");
+    let nextToken = this.tokenizer.checkNext();
+    if (!isNaN(Number(nextToken))) {
+      this.limit = Number(this.tokenizer.getNext());
```

Comment:
do you think it might be an issue if isNaN and Number() allows for decimals?

Reply:
Maybe Math.round() it here so we never run into runtime error when trying to get e.g. the hardest 5.5 cards depending on implementation